clang/lib/AST/ASTImporter.cpp

```diff
@@ -3649,19 +3649,15 @@ class IsTypeDeclaredInsideVisitor
 
 /// This function checks if the function has 'auto' return type that contains
 /// a reference (in any way) to a declaration inside the same function.
```

Comment:
```suggestion
/// This function checks if the given function has a return type that contains
/// a reference (in any way) to a declaration inside the same function.
```

Reply:
Thanks for reminding, fixed in https://github.com/llvm/llvm-project/pull/89096/commits/3f1d714a2aa4b236afe9bb1384be073fb155a2b8